src/App.vue

```diff
@@ -1,38 +1,33 @@
 <script setup lang="ts">
 import { RouterLink, RouterView } from 'vue-router'
 import Sidebar from './components/struture/SideBar.vue'
+import Header from './components/struture/HeaderBar.vue'
 </script>
 
 <template>
-  <Sidebar />
-
-  <div class="content">
-    <header>Projeto Academia</header>
-    <RouterView />
+  <div class="main-container">
+    <Sidebar />
+    <div class="content">
+      <Header />
+      <RouterView />
+    </div>
   </div>
 </template>
 
 <style scoped>
-header {
-  line-height: 1.5;
-  background-color: lightgray;
-  height: 4rem;
+.main-container {
+  display: flex;
+  position: fixed;
+  background-color: #f0f2f5;
```

Comment:
Crie um arquivo de variáveis css e faça um da cor branca, assim, todas as vezes que você desejar trocar a mesma cor, você só vai precisar trocar a variável
```suggestion
  background-color: var(--main-bg-color);
```
ou vai ser 
```suggestion
  background-color: var(--white);
```

depende do que realmente será a variável